lib/RTCSession.js

```diff
@@ -931,7 +944,20 @@ module.exports = class RTCSession extends EventEmitter
           this._localMediaStream = stream;
           if (stream)
           {
-            this._connection.addStream(stream);
+            stream.getTracks().forEach((track) => {
+            let sender = this._connection.addTrack(track, stream);
+            const params = sender.getParameters();
+            params.encodings = params.encodings.map((e) => {
+              if (e.priority) {
+                e.priority = 'high';
+              }
+              if (e.networkPriority) {
+                e.networkPriority = 'high';
+              }
+              return e;
+            });
+            sender.setParameters(params);
+          });
```

Comment:
nit: Since all three spots have the same code, could we put it into a function and remove the duplicated code?